impls/src/adapters/http.rs

```diff
@@ -107,7 +106,7 @@ impl HttpSlateSender {
 
 		// trivial tests for now, but will be expanded later
 		if foreign_api_version < 2 {
-			let report = format!("Other wallet reports unrecognized API format.");
+			let report = "Other wallet reports unrecognized API format.".to_string();
```

Comment:
Some of this is 6 of one, half a dozen of the other kind of stuff. to_string is probably marginally more efficient.